### ISSUE TITLE
Bookmarkable diff between tags/branches [Tigris #71]

### ISSUE THREAD

**Issue by cmpilato** [viewvc/viewvc#21]
## Description

            * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
            *                                                         *
            *   This issue was imported from the SourceForge ViewCVS  *
            *   project's tracker, where it was artifact ID  728869   *
            *                                                         *
            * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
    
    
    ---------------------------------------------------------------------------
        Date: Mon Apr 28 12:03:58 2003
    Username: nobody
    ---------------------------------------------------------------------------
    
    I'd like to have a diff between two branches, for example HEAD and BRANCH_X.
    Today, I can select the two branches in drop-down boxes and get a diff. However,
    the URL contains the revision numbers, so the page will remain the same even if
    the branches are updated.
    
    In short, I need a bookmarkable diff between the latest revisions of two branches.
    
    Gaute Hvoslef Kvalnes
    gaute@verdsveven.com

## Metadata Imported from Tigris (Issue 71)

  * **Creation Date**: 2005-11-14 10:17:47
  * **Reporter**: cmpilato
  * **Subcomponent**: core
  * **Version**: *unspecified*
  * **Milestone**: *unscheduled*
  * **Keywords**: 
  * **Cc**: gnel

### 2006-12-30 16:36:49 by gnel

    This issue may depend or is related to issue 23 where a feature, to get a diff
    of all changes corresponding to one or several changesets, is requested.

### 2009-02-26 07:52:51 by cmpilato

    Moving reviewed-but-not-scheduled issues into a new milestone (to distinguish
    them from unreviewed one).
